summoner-tui/src/Summoner/Tui/Validation.hs

```diff
@@ -105,16 +108,22 @@ showFormError = \case
 -- | Returns list of all invalid fields according to the error.
 errorToInvalidFields :: FormError -> NonEmpty SummonForm
 errorToInvalidFields = \case
-    EmptyFields fields -> fields
-    OneWord fields     -> fields
-    ProjectExist       -> one ProjectName
-    CabalOrStack       -> CabalField :| [StackField]
-    LibOrExe           -> Lib :| [Exe]
+    EmptyFields fields  -> fields
+    OneWord fields      -> fields
+    ProjectExist        -> one ProjectName
+    CabalOrStack        -> CabalField :| [StackField]
+    LibOrExe            -> Lib :| [Exe]
+    PreludePackageValid -> one CustomPreludeName
```

Comment:
Let's rename this constructor. You can that we don't add suffixes likes `Invalid` or `Valid` to the constructors, so more like an indicator or the check name. And semantics are attached later via boolean functions.
```suggestion
    PreludePackage -> one CustomPreludeName
```

Reply:
Thanks! Now also the diff is cleaner :)